modelarmor/modelarmor_test.go

```diff
@@ -96,6 +129,44 @@ func TestCreateModelArmorTemplate(t *testing.T) {
 	}
 }
 
+// TestCreateModelArmorTemplateWithMetadata tests the creation of a template with metadata.
+// Verifies the success message is printed after template creation.
+func TestCreateModelArmorTemplateWithMetadata(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	locationID := testLocation(t)
+	templateID := fmt.Sprintf("test-model-armor-%s", uuid.New().String())
+	templateName := fmt.Sprintf("projects/%s/locations/%s/templates/%s", tc.ProjectID, locationID, templateID)
+
+	var b bytes.Buffer
```

Comment:
nit: I prefer to name the buffer `buf` so that it's easier to find during debugging.